guides/common/modules/proc_customizing-registration-by-using-snippets.adoc

```diff
@@ -0,0 +1,22 @@
+[id="customizing-registration-by-using-snippets"]
+= Customizing registration by using snippets
+
+You can customize the registration process without editing the *Global Registration* template by creating snippets that the template includes if they exist.
+To add custom steps to registration, create one or both of the following snippets:
+
+* `before_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the beginning of registration.
+* `after_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the end of registration.
+
+You have to name the snippets precisely.
+Otherwise, they cannot be recognized.
+
+.Prerequisites
+* Your {Project} account has a role that grants the `view_provisioning_templates`, `create_provisioning_templates`, and `<something to save it>` permissions.
```

Comment:
@stejskalleos I tested the procedure and I was unable to save the new snippet when my account had just `view_provisioning_templates` and `create_provisioning_templates` permissions. Can you please help me to figure out what other permissions are required?

Reply:
(Discussed on a chat.) There's a bug that requires permissions `assign_organizations` and `assign_locations`, which shouldn't be needed for snippets. This PR https://github.com/theforeman/foreman/pull/10078 should fix it.